CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ Wazuh commit: TBD \
 Release report: TBD
 
 ### Added
-
+- Add VDT tests support for ALAS 2022 provider ([#2649](https://github.com/wazuh/wazuh-qa/pull/2903))
```

Comment:
```suggestion
- Add test_providers cases for Amazon Linux 2022. ([#2649](https://github.com/wazuh/wazuh-qa/pull/2903))
```

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/2911/commits/0b47a85d7ee857afef2bd84c5eaa4fb93ddf9f4a